### PR TITLE
Fixes #7

### LINKED ISSUE #7: `em` acts like `rem`
The `em` unit for font sizes doesn't currently respect it's parent's values for font-size.

### PR DESCRIPTION
**Pull request by Galaxtone**
I request that this be merged into dev branch and the old Feature/rem to remd be removed.

### CODE REVIEW

File: src/renderer/components/basic.lua [hunk 1]
```diff
@@ -22,6 +22,23 @@ local function calcSizeBig(text)
   return math.ceil(calcWidth(text) / 2) * 2, math.ceil(font.height / 3) * 3
 end
 
+local function calcTextSize(node)
+  if not node.styles["font-size"] then
+    return 1
```
Comment (incinirate) [inline src/renderer/components/basic.lua]: If an element doesn't have an explicit `font-size`, shouldn't it inherit its parent's size?

Comment (Galaxtone) [reply src/renderer/components/basic.lua]: Ah, right!

**Comment by Galaxtone** [comment]
@incinirate I've fixed the issues! It should be all good!

File: src/renderer/components/basic.lua [hunk 2]
```diff
@@ -22,6 +22,19 @@ local function calcSizeBig(text)
   return math.ceil(calcWidth(text) / 2) * 2, math.ceil(font.height / 3) * 3
 end
 
+local function calcTextSize(node)
+  local size, root = node.styles["font-size"] and node.styles["font-size"]:match("(%d+)(r?)em")
```
Comment (incinirate) [inline src/renderer/components/basic.lua]: What about `0.5em`? `%d+` will only catch integers. Consider `[%d.]+`, and then `size = tonumber(size)` afterwards

Comment (Galaxtone) [reply src/renderer/components/basic.lua]: Fixed.

**Review by incinirate — APPROVED** [review]

**Pull request merged**